app/plugins/hmac-authentication.md

```diff
@@ -35,27 +42,35 @@ $ curl -X POST http://kong:8001/apis/{api}/plugins \
 
 `api`: The `id` or `name` of the API that this plugin configuration will target
 
-You can also apply it for every API using the `http://kong:8001/plugins/` endpoint. Read the [Plugin Reference](/docs/latest/admin-api/#add-plugin) for more information.
+You can also apply it for every API using the `http://kong:8001/plugins/`
+endpoint. Read the [Plugin Reference](/docs/latest/admin-api/#add-plugin)
+for more information.
 
 form parameter                          | default | description
----                                     | ---     | ---
-`name`                                  |         | The name of the plugin to use, in this case: `hmac-auth`
-`config.hide_credentials`<br>*optional* | `false` | An optional boolean value telling the plugin to hide the credential to the upstream API server. It will be removed by Kong before proxying the request
-`config.clock_skew`<br>*optional*       | `300`   | Clock Skew in seconds to prevent replay attacks.
-`config.anonymous`<br>*optional*           | `` | An optional string (consumer uuid) value to use as an "anonymous" consumer if authentication fails. If empty (default), the request will fail with an authentication failure `4xx`
-
+---                                     | --- | ---
+`name`                                  | | The name of the plugin to use, in this case: `hmac-auth`
+`config.hide_credentials`<br>*optional* | `false` | A boolean value telling the plugin to hide the credential to the upstream API server. It will be removed by Kong before proxying the request
+`config.clock_skew`<br>*optional*       | `300` | [Clock Skew][clock-skew] in seconds to prevent replay attacks.
+`config.anonymous`<br>*optional*        | `` | A string (consumer uuid) value to use as an "anonymous" consumer if authentication fails. If empty (default), the request will fail with an authentication failure `4xx`
+`config.validate_request_body`<br>*optional* | `false` | A boolean value telling the plugin to enable body validation
+`config.enforce_headers`<br>*optional*  | `` | A list of headers which client should at least use for http signature creation
+`config.algorithms`<br>*optional*       | `hmac-sha1`,<br>`hmac-sha256`,<br>`hmac-sha384`,<br>`hmac-sha512` | A list of HMAC digest algorithms which user wants to support. Allowed values are `hmac-sha1`, `hmac-sha256`, `hmac-sha384`, and `hmac-sha512`
 ----
 
 ## Usage
 
-In order to use the plugin, you first need to create a consumer to associate one or more credentials to. The Consumer represents a developer using the final service/API.
+In order to use the plugin, you first need to create a consumer to associate
+one or more credentials to. The Consumer represents a developer using the
+final service/API.
```

Comment:
> The Consumer represents a developer using the final service/API.

Consumers are an abstract concept in Kong, not necessarily a developer. I don't think we should make such statements here, but rather leave the topic of what a Consumer is up to the documentation around Consumer itself.